pages/cookbook/nfts.mdx

```diff
@@ -2,8 +2,215 @@
+    customPayload: Cell? = null; //  optional custom data. In most cases should be null
+    forwardAmount: Int as coins; // the amount of nanotons to be sent to the new owner.
+    forwardPayload: Slice as remaining; // optional custom data that should be sent to the new owner.
+}
+
+receive("transfer") {
+    let nftAddress = calculateNFTAddress(self.nftItemIndex, self.nftCollectionAddress, self.nftCode);
+
+    send(SendParameters{
+        to: nftAddress,
+        value: ton("0.1"),
+        body: NFTTransfer{
+            queryId: 42,
+            newOwner: address("NEW_OWNER_ADDRESS"), // NOTE: Modify it to your liking
+            responseDestination: myAddress(),
+            customPayload: null,
+            forwardAmount: 1,
+            forwardPayload: rawSlice("F"), // precomputed beginCell().storeUint(0xF, 4).endCell().beginParse()
+        }.toCell(),
+    });
+}
+```
+
+## Get NFT static info
+
+Note, that TON Blockchain does not allow contracts to call each other [getters](https://docs.tact-lang.org/book/contracts#getter-functions) like Ethereum does.
+In order to receive data from another contract, you must exchange messages.
+
+```tact
+message(0x2fcb26a2) NFTGetStaticData {
+    queryId: Int as uint64;
+}
+
+message(0x8b771735) NFTReportStaticData {
+    queryId: Int as uint64;
+    index: Int as uint256;
+    collection: Address;
+}
+
+receive("get static data") {
+    let nftAddress = address("[NFT_ADDRESS]");
+    send(SendParameters{
+        to: nftAddress,
+        value: ton("0.1"),
+        body: NFTGetStaticData{
+            queryId: 42,
+        }.toCell(),
+    });
+}
+
+receive(msg: NFTReportStaticData) {
+    let expectedNftAddress = calculateNFTAddress(msg.index, msg.collection, self.nftCode);
+    require(expectedNftAddress == sender(), "NFT contract is not the sender");
+
+    // Save nft static data or do something
+}
+```
+
+## Get NFT royalty params
+
+NFT royalty params are described [here](https://github.com/ton-blockchain/TEPs/blob/master/text/0066-nft-royalty-standard.md).
+
+```tact
+message(0x693d3950) NFTGetRoyaltyParams {
+    queryId: Int as uint64;
+}
+
+message(0xa8cb00ad) NFTReportRoyaltyParams {
+    queryId: Int as uint64;
+    numerator: Int as uint16;
+    denominator: Int as uint16;
+    destination: Address;
+}
+
+receive("get royalty params") {
+    send(SendParameters{
+        to: self.nftCollectionAddress,
+        value: ton("0.1"),
+        body: NFTGetRoyaltyParams{
+            queryId: 42,
+        }.toCell(),
+    });
+}
+
+receive(msg: NFTReportRoyaltyParams) {
+    require(self.nftCollectionAddress == sender(), "NFT collection contract is not the sender");
+
+    // Do something with msg
+}
+```
+
+## NFT Collection methods
+
+These methods are not described in any standard, but are commonly used in NFT [contracts](https://github.com/ton-blockchain/token-contract/blob/main/nft/nft-collection.fc). Note that they can only be used by NFT owners.
```

Comment:
Since these methods aren't described by TEP-62, they can be implemented in a different way, so we should either remove this part from docs, or mention that this is just an example of sending this kind of a message that will only work with some existing implementation (put a link to it)

Reply:
rewrited and added callout to attract attention. Resolved